CONTRIBUTING.md

```diff
@@ -87,11 +87,19 @@ The CockroachDB Jekyll theme can auto-generate a page-level table of contents li
 
 #### TOC Levels
 
-By default, a page TOC includes h2 and h3 headers. To limit a page TOC to h2 headers only, set `toc_not_nested: true` in the page's front-matter. 
+By default, a page TOC includes h2 and h3 headers. To limit a page TOC to h2 headers only, set `toc_not_nested: true` in the page's front-matter.
 
 ### Sidebar
 
-The [`_data/sidebar_doc.yml`](_data/sidebar_doc.yml) file controls which pages appear in the docs sidebar. If you're adding a page that you think should appear in the sidebar, please mention this in your pull request.
+The [`_data/sidebar_doc.yml`](_data/sidebar_doc.yml) YAML file controls which pages appear in the docs sidebar. If you're adding a page that you think should appear in the sidebar, please mention this in your pull request.
+
+For each item in the sidebar, there are three possible YAML fields:
+
+Field | Description
+------|------------
+`title` | The page title, as displayed in the sidebar. This can be different from the actual page title.
+`url` | The URL of the page to link to, formatted as `/<page-name>.html`, e.g., `/learn-cockroachdb-sql.html`.
+`alternate_urls` | The URLs of other pages that should highlight this title in the sidebar, as a comma-separated list, e.g., `[/build-a-go-app-with-cockroachdb-pg.html, /build-a-go-app-with-cockroachdb-gorm.html]`.
```

Comment:
Good point. I wonder whether we should go with that other list format by default, since that's how we're handling lists throughout the rest of the document... I'll play with it a bit.